solutions/devicesimulation-nohub/single-vm/docker-compose.yml

```diff
@@ -62,6 +62,8 @@ services:
       - PCS_CORS_WHITELIST
       - PCS_SUBSCRIPTION_DOMAIN
       - PCS_SUBSCRIPTION_ID
+      - PCS_AAD_SP_CLIENT_ID
```

Comment:
I am not sure about if the service principal id should be used when authenticate AAD to get AAD token, usually we use AAD application id + application secret to get token. And you also need AAD tenant id in the API. you can reuse `PCS_WEBUI_AUTH_AAD_APPID` and `PCS_WEBUI_AUTH_AAD_TENANT`.